code/_onclick/hud/parallax/parallax.dm

```diff
@@ -11,6 +11,10 @@
 	for(var/atom/movable/screen/plane_master/parallax as anything in get_true_plane_masters(PLANE_SPACE_PARALLAX))
 		parallax.unhide_plane(screenmob)
 
+	if(!C.parallax_rock)
```

Comment:
```suggestion
	if(isnull(C.parallax_rock))
```

Reply:
I TRIED TO WARD AGAINST YOUR EVIL WAYS, BEAST